pkg/appgw/requestroutingrules_test.go

```diff
@@ -0,0 +1,115 @@
+// -------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// --------------------------------------------------------------------------------------------
+
+package appgw
+
+import (
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2018-12-01/network"
+	"github.com/Azure/go-autorest/autorest/to"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var expectedRedirectID = "/subscriptions/" + testFixturesSubscription +
```

Comment:
How about using this instead of constructing ?
```go
agw := Identifier{
    SubscriptionID: testFixturesSubscription,
    ResourceGroup:  testFixtureResourceGroup
	AppGwName:      testFixtureAppGwName
}
configName := generateSSLRedirectConfigurationName(testFixturesNamespace, testFixturesName)
expectedRedirectID := agw.redirectConfigurationID(configName)
```

Reply:
Works for me!